src/fare-contracts/details/DetailsContent.tsx

```diff
@@ -170,6 +179,18 @@ export const DetailsContent: React.FC<Props> = ({
             />
           </GenericSectionItem>
         )}
+        {benefits && (
+          <GenericSectionItem style={styles.benefitSection}>
+            <ThemeText type="body__secondary" color="secondary">
+              {t(FareContractTexts.label.indludedBenefits)}
+            </ThemeText>
+            <BenefitTiles
+              benefits={benefits}
+              onNavigateToMap={onNavigateToMap}
+              interactiveColor="interactive_2"
+            />
+          </GenericSectionItem>
+        )}
```

Comment:
To follow the pattern from the other benefit stuff, could this be a section item? We have `MobilityBenefitsInfoSectionItem`, maybe this could be `MobilityBenefitsActionsSectionItem`.

Reply:
Sure, updated it now.